lib/ares.rb

```diff
@@ -7,6 +7,16 @@
 require 'ico-validator'
 
 module Ares
+  # Default timeout for Ares requests
+  DEFAULT_TIMEOUT = 5
+
+  @@timeout = DEFAULT_TIMEOUT
```

Comment:
You are replying to comment from houndci (rubocop behind). Try to find solution in ruby styleguide.